FertigMelder/module.php

```diff
@@ -18,14 +18,15 @@ public function Create()
 
         if (!IPS_VariableProfileExists('FM.Status')) {
             IPS_CreateVariableProfile('FM.Status', 1);
-            IPS_SetVariableProfileValues('FM.Status', 0, 2, 1);
             IPS_SetVariableProfileAssociation('FM.Status', 0, $this->Translate('Off'), 'Sleep', -1);
             IPS_SetVariableProfileAssociation('FM.Status', 1, $this->Translate('Running'), 'Motion', -1);
             IPS_SetVariableProfileAssociation('FM.Status', 2, $this->Translate('Done'), 'Ok', -1);
         }
 
-        $this->RegisterVariableInteger('Status', 'Status', 'FM.Status');
-        $this->RegisterVariableBoolean('Active', 'Active', '~Switch');
+        IPS_SetVariableProfileValues('FM.Status', 0, 0, 0);
```

Comment:
Hier wäre noch ein Kommentar schön, warum das herausgezogen wurde (-> Anpassung von Legacy-Profilen)

Reply:
Der Text erörtert nicht wirklich den meiner Meinung nach relevanten Punkt: Das war vorher ja anders und soll daher für existierende gefixt werden. Also ist daher für mich der Punkt des fixen von Legacy-Versionen sehr relevant. Das steckt nur dezent in "existing profiles" drin.